### PR TITLE
Fix activation stdout

### PR DESCRIPTION
**Pull request by benthomasson**
Depends on #71 

Activation page was really slow due to fast stdout updates.   This changes the stdout updates to only listen when on the stdout route.

### CODE REVIEW

**Comment by benthomasson** [comment]
@lgalis please review

File: ui/src/app/Activation/activation-stdout.tsx [hunk 1]
```diff
@@ -8,12 +9,49 @@ import {
   StackItem,
 } from '@patternfly/react-core';
 import {useIntl} from "react-intl";
+import {getServer} from "@app/utils/utils";
 import {renderActivationTabs} from "@app/Activation/Activation";
 import Ansi from "ansi-to-react";
 import {ActivationType} from "@app/Activations/Activations";
 
-const ActivationStdout: React.FunctionComponent<{activation: ActivationType, stdout: string[]}> = ({activation, stdout}) => {
+const endpoint3 = 'http://' + getServer() + '/api/activation_instance_logs/?activation_instance_id=';
```
Comment (cutwater) [inline ui/src/app/Activation/activation-stdout.tsx:17]: Can you please use better naming instead of `endpoint1, endpoint2, endpoint3`?

**Review by cutwater — CHANGES_REQUESTED** [review]

**Review by lgalis — APPROVED** [review]

File: ui/src/app/Activation/activation-stdout.tsx (hunk 1 above)
Comment (lgalis) [reply ui/src/app/Activation/activation-stdout.tsx:17]: @cutwater - I will refactor all the endpoints in a separate PR - it's on my ToDo list

Comment (cutwater) [reply ui/src/app/Activation/activation-stdout.tsx:17]: Ok

**Pull request merged**